rasa/server.py

```diff
@@ -998,7 +998,14 @@ async def add_message(request: Request, conversation_id: Text) -> HTTPResponse:
 
         try:
             async with app.agent.lock_store.lock(conversation_id):
-                tracker = await app.agent.log_message(user_message)
+                # cf. processor.handle_message (ignoring prediction loop run)
+                tracker = await app.agent.processor.log_message(
+                    user_message, should_save_tracker=False
+                )
+                tracker = await app.agent.processor._run_action_extract_slots(
```

Comment:
Would it be a good idea to make this method public with a simple docstring? Originally I made it private thinking it won't be used outside of the `MessageProcessor`, but here we are 😄 
Same consideration for `_save_tracker` too?

Reply:
oh good point, yes

Reply:
Yeah might mean changing the method usage to public in several places in `MessageProcessor` 😄